### PR TITLE
Feature checkpoint

### PR DESCRIPTION
**Pull request by cGoGo**

### CODE REVIEW

File: que-se-espera/desarrollo-de-carreras.md [hunk 1]
```diff
@@ -1,3 +1,103 @@
...
+### ¿Cómo inicio mi evaluación?
+
+Esta herramienta permitirá autoevaluarse cuando y cuantas veces quieras. Sin embargo, cada **tres meses** podrás solicitar resultados de manera formal, es decir, completando cada etapa del Checkpoint. Aquí te indicamos **las etapas**:
+
+1. Crear una hoja de cálculo a partir de una plantilla llamada, Checkpoint [Nombre] Designer/Developer Template. Modifica el nombre de este nuevo archivo creado, con tu nombre, por ejemplo “Checkpoint [Juan Pérez] Developer Template".
+
+![cómo crear un template](images/ejemplo_crear_template.png)
+
+2. Abre la planilla COMPARTIDA con los ejemplos, escala y pilares; para que tengas las referencias y puedas dejar los comentarios que estimes convenientes:
+- LINK 1
```
Comment (pcortez) [inline que-se-espera/desarrollo-de-carreras.md]: developer: https://docs.google.com/spreadsheets/d/18sWYoli2V2QwkbtpBiVRmQXT05mdi3OwnRBzFRjpf_c/edit?usp=sharing

File: que-se-espera/desarrollo-de-carreras.md [hunk 2]
```diff
@@ -1,3 +1,103 @@
...
+- LINK 2
```
Comment (pcortez) [inline que-se-espera/desarrollo-de-carreras.md]: product designer: https://docs.google.com/spreadsheets/d/1dW4IQYaqe_foYkSPQ6FE2L1a4iirDhtpWu2UlFdZELU/edit?usp=sharing

File: que-se-espera/desarrollo-de-carreras.md (hunk 1 above)
Comment (cGoGo) [reply que-se-espera/desarrollo-de-carreras.md]: Agregado

File: que-se-espera/desarrollo-de-carreras.md (hunk 2 above)
Comment (cGoGo) [reply que-se-espera/desarrollo-de-carreras.md]: Agregado

**Pull request merged**